backend/src/lib.rs

```diff
@@ -131,7 +131,7 @@ fn unmap_paulis(state: &mut SimulatorState, combined_list: Vec<(Pauli, usize)>)
             Pauli::X => state.sim.h(qubit),
```

Comment:
Can we rename the `(un)map_paulis` functions to be more descriptive and maybe a doc comment.